### ISSUE TITLE
Identation for argument comments

### ISSUE THREAD

**Issue by webmake** [squizlabs/PHP_CodeSniffer#1479]
Single line comments for unknown reason is changed to 3 lines documentation. Even more, that identation is quite strange:
before
```
    /** @var array */
    protected $hidden = [
        'created_at',
        'updated_at',
    ];
```
after
```
    /**
 * @var array 
*/
    protected $hidden = [
        'created_at',
        'updated_at',
    ];
```

Full examples
```
<?php
class MyClass
{
    /** @var array */
    protected $hidden = [
        'created_at',
        'updated_at',
    ];
}
```
```
<?php
class MyClass
{
    /**
 * @var array 
*/
    protected $hidden = [
        'created_at',
        'updated_at',
    ];
}
```
output
```
$ ./vendor/bin/phpcbf  Entities/MyClass.php --no-patch
Changing into directory /var/www/Entities
Processing MyClass.php [PHP => 37 tokens in 9 lines]... DONE in 16ms (2 fixable violations)
        => Fixing file: 0/2 violations remaining [made 2 passes]... DONE in 9ms 
        => File was overwritten
Fixed 1 files
Time: 523ms; Memory: 4Mb

```
but running again and again nothing changes more, so it is final version with weird spacing
```
Changing into directory /var/www/Entities
Processing MyClass.php [PHP => 41 tokens in 11 lines]... DONE in 16ms (0 fixable violations)
Fixed 0 files
Time: 578ms; Memory: 4Mb
```

**Comment by gsherwood** [comment]
It looks like you are using the PEAR standard, which didn't include the specific sniff and auto-fix to align doc comments properly. I've now added that and it will be released in 3.0.1.

For this sample file:
```php
<?php
class Foo {
    /** @var array */
    protected $hidden = [
        'created_at',
        'updated_at',
    ];
}
```
The fixer will now produce this diff when using the PEAR standard, which includes the doc comment alignment:
```diff
--- temp.php
+++ PHP_CodeSniffer
@@ -1,6 +1,9 @@
 <?php
-class Foo {
-    /** @var array */
+class Foo
+{
+    /**
+     * @var array
+     */
     protected $hidden = [
         'created_at',
         'updated_at',
```
